source/installation/deploy-minio-standalone.rst

```diff
@@ -1,63 +1,45 @@
-objects as-is without any :ref:`erasure coding <minio-erasure-coding>`.
+The procedures on this page cover deploying MinIO in a Single-Node Single-Drive (SNSD) configuration for early development and evaluation.
+This mode was previously called :guilabel:`Standalone Mode` or 'filesystem' mode.
 
-For extended development or production environments, *or* to access
-:ref:`advanced MinIO functionality <minio-installation-comparison>` deploy MinIO
+For extended development or production environments, deploy MinIO
 in :guilabel:`Distributed Mode`. See :ref:`deploy-minio-distributed` for more
 information.
 
 Pre-Existing Data
 -----------------
 
-When starting a new standalone MinIO server, the storage path may have existing data already grouped into directories.
-MinIO displays this data as buckets and objects in the MinIO deployment.
+Starting with :minio-release:`RELEASE.2022-06-02T02-11-04Z`, MinIO implements a zero-parity erasure coded backend for single-node single-drive deployments.
+This feature allows access to :ref:`erasure coding dependent features <minio-erasure-coding>` without the requirement of multiple drives.
+The specified data volume or folder *must* be empty *or* only contain files generated by a previous |SNSD| deployment.
 
-- Top-level folders at the starting path become MinIO buckets.
-- Files in folders within the starting path become objects within the MinIO buckets.
-- Files in the top-level starting path do not display in MinIO.
+Use an earlier version of MinIO to retain the legacy filesystem-access behavior.
 
-Consider a starting path with the following structure:
+.. admonition:: MinIO Avoids Root Disks
+   :class: important
 
-.. code-block:: 
-   
-   /data
-      file.txt
-      /foo
-         file2.txt
-      /bar
-         file3.txt
+   MinIO by default avoids using the root OS disk as part of startup and returns an error if the specifid path is part of the root disk.
 
-When you deploy the server with a starting path of ``/data``, MinIO displays the data as follows:
-
-- Two buckets, ``foo`` and ``bar``
-- ``file2.txt`` displays in the ``foo`` bucket 
-- ``file3.txt`` displays in the ``bar`` bucket
-- ``file.txt`` does not display in MinIO anywhere, but remains available through the file system
-
-On a standalone deployment with a single drive, you can manage buckets and objects with the :ref:`Console <minio-console>`, :ref:`command line client <minio-client>`, or create and delete files and folders directly in the starting path folders.
+   You can specify the environment variable ``MINIO_CI_CD=1`` to allow using root disks.
 
+   This variable is intended primarily for CI/CD environments or local testing/evaluation.
+   MinIO does not support usage outside of those specific use cases.
 
 .. _deploy-minio-standalone:
 
-Deploy Standalone MinIO on Baremetal
-------------------------------------
+Deploy Single-Node Single-Drive MinIO on Baremetal
+--------------------------------------------------
 
```

Comment:
Question: Can SNSD be on something other than Baremetal?
If not, do we need "on Baremetal" in the heading?

Reply:
I think as part of the IA switch we would drop any reference to Baremetal entirely, and can do that safely here since its the title and not the URL